ros2doctor/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2doctor</name>
+  <version>0.0.0</version>
+  <description>A command line tool to check potential issues in ROS2 system</description>
+  <maintainer email="clairewang@openrobotics.org">Claire Wang</maintainer>
+  <license>Apache License 2.0</license>
+
+  <depend>ros2cli</depend>
+
+  <exec_depend>python3-numpy</exec_depend>
+  <exec_depend>python3-rosdistro</exec_depend>
```

Comment:
This is not going to work. `python3-rosdistro` will conflict with `python-rosdistro` which would result in uninstalling ROS 1 stuff when installing this Debian package.

Instead this package should depend on `python3-rosdistro-modules` only (similar to `python3-catkin-pkg-modules`). This will still require a new `rosdep` key (similar to this one: https://github.com/ros/rosdistro/blob/0ea30fed0bcb564b9e4d5b2d2efd8783e8e3afa9/rosdep/python.yaml#L4913).

Reply:
Does something like this look right? 
`python3-rosdistro-modules:`
   `debian: [python3-rosdistro-modules]`
   `fedora: [python3-rosdistro]`
   `gentoo: [dev-python/rosdistro]`
   `openembedded: [python3-rosdistro@meta-ros]`
   `rhel: ['python%{python3_pkgversion}-rosdistro']`
   `ubuntu: [python3-rosdistro-modules]`
I'm gonna add this to `python.yaml` in my forked repo, test then create a pull request.